Sources/JNI/JavaParameterConvertible+Primitives.swift

```diff
@@ -25,32 +25,32 @@ extension Bool: JavaParameterConvertible, JavaInitializableFromMethod, JavaIniti
 }
 
 
-// Int
+// JavaInt
 
-extension Int: JavaParameterConvertible, JavaInitializableFromMethod, JavaInitializableFromField {
+extension JavaInt: JavaParameterConvertible, JavaInitializableFromMethod, JavaInitializableFromField {
     public static var asJNIParameterString = "I"
 
     public func toJavaParameter() -> JavaParameter {
         return JavaParameter(int: JavaInt(self))
     }
 
-    public static func fromStaticField(_ fieldID: JavaFieldID, of javaClass: JavaClass) throws -> Int {
+    public static func fromStaticField(_ fieldID: JavaFieldID, of javaClass: JavaClass) throws -> JavaInt {
         let result = try jni.GetStaticIntField(of: javaClass, id: fieldID)
-        return Int(result)
+        return JavaInt(result)
```

Comment:
I think these can be directly returned now `return try jni.GetStaticIntField(of: javaClass, id: fieldID)`